### PR TITLE
libimobiledevice: update to their official repo URL, and use https.

### PR DESCRIPTION
**Pull request by cbowns**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

While I [wait for a new tag from `libimobiledevice`](https://github.com/libimobiledevice/libimobiledevice/issues/340), I wanted to update the `HEAD` formula to point to:
- A secure server
- A URL which matches that provided by the project. (The git.sukimashita.com URL is weird.)

### CODE REVIEW

**Comment by cbowns** [comment]
- This formula currently lacks a test, and fails the `audit`.
- I'm reading the contribution guidelines now.

File: Formula/libimobiledevice.rb [hunk 1]
```diff
@@ -36,4 +36,10 @@ def install
                           "--without-cython"
     system "make", "install"
   end
+
+  test do
+    # Many of libimobiledevice's actions depend on a plugged-in iDevice.
+    # A bad test is better than none?
+    system "idevicedate", "--help"
```
Comment (MikeMcQuaid) [inline Formula/libimobiledevice.rb]: `system "#{bin}/idevicedate", "--help"`

**Comment by MikeMcQuaid** [comment]
yep, bad test is definitely better than none 👍

File: Formula/libimobiledevice.rb [hunk 2]
```diff
@@ -36,4 +36,10 @@ def install
                           "--without-cython"
     system "make", "install"
   end
+
+  test do
+    # Many of libimobiledevice's actions depend on a plugged-in iDevice.
+    # A bad test is better than none?
```
Comment (DomT4) [inline Formula/libimobiledevice.rb]: Can cut this comment as well.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/08fb3d6f0f1ccb8e17339afb45e3f9f27384523c. Thank you for your contribution to Homebrew @cbowns; we appreciate it! :smiley_cat:

**Pull request merged**